test/e2e/data/infrastructure-cloudstack/v1beta1/bases/cluster-with-kcp.yaml

```diff
@@ -49,8 +49,58 @@ spec:
         name: '{{ local_hostname }}'
         kubeletExtraArgs:
           provider-id: cloudstack:///'{{ ds.meta_data.instance_id }}'
+        ignorePreflightErrors:
+          - DirAvailable--etc-kubernetes-manifests
     preKubeadmCommands:
       - swapoff -a
+    files:
```

Comment:
Should we make this kube-vip static-pod conditional? so the customer using external load balancer can choose not using kube-vip.

Reply:
This yaml file is only for e2e testing.